examples/send-and-receive-emails/send_and_receive_emails.bal

```diff
@@ -42,21 +42,21 @@ public function main() {
     }
 
     // Create the client with the connection parameters, host, username, and
-    // password. An error is received in failure. Default port number `995` is
+    // password. An error is received in a failure. The default port number `995` is
     // used over SSL with these configurations.
     email:PopClient|email:Error popClient = new ("pop.email.com",
```

Comment:
@Maninda shall we remove the specific error handling code and use "check" expressions here, while returning "error?" from main. This would make the example shorter, and clearer on the main use case.

Reply:
Created a new PR fixing this issue https://github.com/ballerina-platform/ballerina-distribution/pull/843